src/composables/LetoModelizerApi.js

```diff
@@ -42,7 +42,8 @@ export async function getUserRoles(userId, sessionToken) {
 
 /**
  * Get all libraries.
- * @returns {Promise<object[]>} Return an array of libraries.
+ * @returns {Promise<object[]>} Return an array of libraries on success
```

Comment:
Still not good.
Correct sentence is something like this: 
`Promise with the logged-in user information on success otherwise an error.`

Reply:
done